server/package.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
 - directory >= 1.3 && <1.4
 - hashable >=1.2 && <1.3
 - http-types >=0.9 && <0.10
+- jwt >= 0.7 && <0.8
```

Comment:
Doesn't `<0.8` pick the latest version possible smaller than 0.8? Is `>=0.7` redundant?